docs/studios/unity/tutorials/github_actions.md

```diff
@@ -0,0 +1,309 @@
+
+```yaml
+#file: .github/workflows/qa-bot.yml
+
+name: Regression Games Bots QA 🤖
+
+on: [push]
+
+# See https://game.ci/docs/github/test-runner and RG_DOCS for more info
+jobs:
+  build:
+    name: Build my project ✨
+    runs-on: ubuntu-latest
+    permissions: write-all # This is to allow writing results into the artifacts section of GitHub Actions
+    steps:
+          
+      # Checkout (without LFS)
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      # Git LFS
+      - name: Create LFS file list
+        run: git lfs ls-files -l | cut -d' ' -f1 | sort > .lfs-assets-id
+
+      - name: Restore LFS cache
+        uses: actions/cache@v3
+        id: lfs-cache
+        with:
+          path: .git/lfs
+          key: ${{ runner.os }}-lfs-${{ hashFiles('.lfs-assets-id') }}
+
+      - name: Git LFS Pull
+        run: |
+          git lfs pull
+          git add .
+          git reset --hard
+
+      # Cache
+      - uses: actions/cache@v3
+        with:
+          path: Library
+          key: Library-${{ hashFiles('Assets/**', 'Packages/**', 'ProjectSettings/**') }}
+          restore-keys: |
+            Library-
+
+      # Test
+      # See https://game.ci/docs/github/test-runner for more information
+      - name: Run tests
+        uses: game-ci/unity-test-runner@v2
+        env:
+          UNITY_LICENSE: ${{ secrets.UNITY_LICENSE }}
+        with:
+          customParameters: -RG_BOT ${{ secrets.RG_BOT }} -RG_API_KEY ${{ secrets.RG_API_KEY }} -RG_HOST ${{ secrets.RG_HOST }}
+          githubToken: ${{ secrets.GITHUB_TOKEN }}
+
+      # Output
+      - uses: actions/upload-artifact@v3
+        with:
+          name: Build
+          path: build
+```
+
+:::info
+
+Note that this will actually run all of your tests, not just the QA Bot test. To configure this workflow to only
+run the QA test, see the [Unity Test Runner Command Line docs](https://docs.unity3d.com/Packages/com.unity.test-framework@2.0/manual/reference-command-line.html),
+which has filter options that can be passed within `customParameters` within the `Run tests` step above.
+
+:::
+
+## Configure GitHub Actions
+
+The final piece on the GitHub actions side is to configure the secrets that are used in the workflow file above. Visit the secrets
+page for your project by going to your **Repo** > **Settings** > **Secrets and variables** > **Actions**. Add the following secrets
+by clicking the green **New repository secret** button.
+
+1. `RG_HOST` - `https://play.regression.gg`
+2. `RG_API_KEY` - Your API key from Regression Games, which can be found in your [account settings](https://play.regression.gg/account).
+                  Full instructions for this can be found in the [Authentication docs.](studios/api_reference/authentication).
+3. `RG_BOT` - The ID of the bot you want to run. Within the [Bot Manager page](https://play.regression.gg/bots), you can see the ID
```

Comment:
This reads a bit awkwardly. Suggest rewording to something like "The ID of the bot you want to run. You can find this through the bot's details within the Regression Games Bot Manager." (then the image clarifies the _where_)